apps/prairielearn/elements/pl-sketch/pl-sketch.mustache

```diff
@@ -0,0 +1,43 @@
+<div class="pl-sketch" id="pl-sketch-{{uuid}}">
+    <div class="pl-sketch-controls" style="width: {{width}}px">
+        <div class="pl-sketch-control-group">
+            <div class="pl-sketch-control" id="undo-{{uuid}}"><div class="fa fa-undo fa-2x"></div></div>
+            <div class="pl-sketch-control" id="redo-{{uuid}}"><div class="fa fa-redo fa-2x"></div></div>
+            <div class="pl-sketch-control" id="pen-{{uuid}}"><div class="fa fa-pen fa-2x"></div></div>
+            <div class="pl-sketch-control" id="eraser-{{uuid}}"><div class="fa fa-eraser fa-2x"></div></div>
+        </div>
+        <div class="pl-sketch-control-group">
+            <div class="pl-sketch-setting" id="color-picker-{{uuid}}">
+                <div class="pl-sketch-control pl-sketch-color-icon" id="color" style="background-color: #000000"></div>
+                <div class="pl-sketch-popup pl-sketch-hidden" id="color-popup-{{uuid}}">
+                    <div class="pl-sketch-circle" style="background-color: #FFEB3B"></div>
+                    <div class="pl-sketch-circle" style="background-color: #FF9800"></div>
+                    <div class="pl-sketch-circle" style="background-color: #F44336"></div>
+                    <div class="pl-sketch-circle" style="background-color: #E91E63"></div>
+                    <div class="pl-sketch-circle" style="background-color: #9C27B0"></div>
+                    <div class="pl-sketch-circle" style="background-color: #3F51B5"></div><br>
+                    <div class="pl-sketch-circle" style="background-color: #00BCD4"></div>
+                    <div class="pl-sketch-circle" style="background-color: #4CAF50"></div>
+                    <div class="pl-sketch-circle" style="background-color: #795548"></div>
+                    <div class="pl-sketch-circle" style="background-color: #E0E0E0"></div>
+                    <div class="pl-sketch-circle" style="background-color: #616161"></div>
+                    <div class="pl-sketch-circle" style="background-color: #000000"></div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <canvas class="pl-sketch-canvas" id="pl-sketch-canvas-{{uuid}}" data-width="80%" data-height="80%"></canvas>
+    <input type="hidden" id="pl-sketch-input-{{uuid}}" name="sketches-{{uuid}}" value="{{{sketchpad_json}}}" />
```

Comment:
You probably shouldn't use the UUID for a name here. Since UUID is created on render, this means every time you submit a new UUID is used, which means the name will be different in every call. You could change the Python code to generate UUID in `prepare` and save it in `params`, but I believe it makes more sense for you to have an `answers-name` parameter, for consistency with other elements that use the same model.